Added lots of things into the section Gaming (#353)
myfreeweb/awesome-gtk: README.md

  - [System Monitoring and Info](#system-monitoring-and-info)
  - [Task Scheduling](#task-scheduling)
- [Gaming](#gaming)
  - [Action, Platformer, Jump 'n' run](#Action-Platformer-Jump-n-run)

>   - [Adventure](#adventure)

Reviewer: ```suggestion
```
Remove empty category

  - [Board and Card Games](#board-and-card-games)
  - [Puzzles and Quizzes](#puzzles-and-quizzes)

>   - [Role-playing](#role-playing)

Reviewer: ```suggestion
```

Remove empty category

- [An Anime Game launcher](https://github.com/an-anime-team/an-anime-game-launcher) - Genshin Impact launcher for Linux with telemetry disabling `#rust` `#gtk4` `#relm4` `#libadwaita`.
- [Cartridges](https://github.com/kra-mo/cartridges) - Game launcher with Steam, Lutris, Heroic, Legendary, Bottles, itch and RetroArch library import `#python` `#gtk4` `#libadwaita`.

> - [DeSmuME](https://github.com/TASEmulators/desmume) - Nintendo DS emulator `#c++` `#gtk3`.

Reviewer: ```suggestion
- [DeSmuME](https://github.com/TASEmulators/desmume) - Cross-platform Nintendo DS emulator `#c++` `#gtk3`.
```

> - [Epic Asset Manager](https://github.com/AchetaGames/Epic-Asset-Manager) - An unofficial client to install Unreal Engine, download and manage purchased assets, projects, plugins and games from the Epic Games Store `#rust` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Epic Asset Manager](https://github.com/AchetaGames/Epic-Asset-Manager) - Unofficial client to install Unreal Engine, download and manage purchased assets, projects, plugins and games from the Epic Games Store `#rust` `#gtk4` `#libadwaita`.
```

- [Gameeky](https://github.com/tchx84/gameeky) - Application to create and play games without any code for young learners and educators `#python` `#gtk4` `#libadwaita`.
- [Keypunch](https://github.com/bragefuglseth/keypunch) - Keyboard typing test like monkeytype `#rust` `#gtk4` `#libadwaita` `#gnome`.
- [Lutris](https://lutris.net) - Game launcher covering most gaming systems `#python` `#gtk3`.

> - [Snes9x](https://github.com/snes9xgit/snes9x/tree/master) - Portable Super Nintendo Entertainment System (TM) emulator `#c++` `#gtk3`.

Reviewer: ```suggestion
- [Snes9x](http://www.snes9x.com) - Cross-platform portable Super Nintendo Entertainment System (TM) emulator `#c++` `#gtk3`.
```

> ### Action, Platformer, Jump 'n' run

Reviewer: ```suggestion
### Action, Platformer, Jump 'n' run

```

> - [Gnome Robots](https://gitlab.gnome.org/GNOME/gnome-robots) - a game where you try and avoid a band of marauding robots which are, for no adequately explained reason, trying to kill you `#rust` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [GNOME Robots](https://gitlab.gnome.org/GNOME/gnome-robots) - Game where you try to avoid a band of marauding robots which are trying to kill you `#rust` `#gtk4` `#libadwaita`.
```

### Adventure

### Board and Card Games

- [2048](https://gitlab.gnome.org/GNOME/gnome-2048) - A 2048-like game for GNOME `#vala` `#gtk3`.
- [Chance](https://flathub.org/apps/dev.zelikos.rollit) - Roll the Dice application that rolls a six-sided dice by default and allows custom dice of up to 999 sides `#rust` `#gtk4` `#libadwaita`.
- [Convolution](https://gitlab.gnome.org/bazylevnik0/convolution) - Logic game about escaping from a maze `#gjs` `#javascript` `#gtk4` `#libadwaita`.

> - [Crossword Editor (Crosswords)](https://gitlab.gnome.org/jrb/crosswords) - Crossword player and editor for games of crosswords with squpport for shaped and colors crosswords and `.ipuz`, `.jpuz`, `.xd`, and `.puz` files `#c` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Crossword Editor (Crosswords)](https://gitlab.gnome.org/jrb/crosswords) - Crossword player and editor for games of crosswords with support for shaped and color crosswords and `.ipuz`, `.jpuz`, `.xd`, and `.puz` files `#c` `#gtk4` `#libadwaita`.
```

- [Flood It](https://flathub.org/apps/io.github.tfuxu.floodit) - Game with the simple premise of flooding the entire board with one color in the least amount of moves possible `#go` `#gtk4` `#libadwaita`.

> - [goban](https://gitlab.com/gjask/goban) - A Go/Weiqi/Baduk board and SGF editor for the GNOME desktop `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Goban](https://gitlab.com/gjask/goban) - Go/Weiqi/Baduk board and SGF editor for the GNOME desktop `#python` `#gtk4` `#libadwaita`.
```

- [Libellus](https://libellus.hummdudel.de) - Tab-based wiki/Lexicon for DnD 5th edition `#gjs` `#javascript` `#gtk4` `#libadwaita`.
- [Mahjongg](https://gitlab.gnome.org/GNOME/gnome-mahjongg) - Solitaire (one player) version of the classic Eastern tile game, Mahjongg `#vala` `#gtk4` `#libadwaita` .
- [Ultimate Tic Tac Toe](https://github.com/Nokse22/ultimate-tic-tac-toe) - [Ultimate Tic Tac Toe](https://en.wikipedia.org/wiki/Ultimate_tic-tac-toe) to play with friends or against an AI `#python` `#gtk4` `#libadwaita`.

### Puzzles and Quizzes

> - [Hex Colordle GTK](https://gitlab.com/Krafting/hexcolordle-gtk) - A game where you need to find a color by guessing its hex code `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Hex Colordle GTK](https://gitlab.com/Krafting/hexcolordle-gtk) - Game where you need to find a color by guessing its hex code `#python` `#gtk4` `#libadwaita`.
```

- [Multiplication Puzzle](https://gitlab.gnome.org/mterry/gmult) - Simple game inspired by the multiplication game from emacs `#vala` `#gtk4` `#libadwaita`.

> - [PedantiK](https://gitlab.com/Krafting/pedantik) - Pédantix Clone `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [PedantiK](https://gitlab.com/Krafting/pedantik) - Game where players try to find the words to fill a Wikipedia article (clone of Pédantix)  `#python` `#gtk4` `#libadwaita`.
```

- [SemantiK](https://gitlab.com/Krafting/semantik-gtk) - Word-game where you need to find a secret word `#python` `#gtk4` `#libadwaita`.

> - [Trivia Quiz](https://github.com/Nokse22/trivia-quiz) - This app let's you respond to an endless stream of questions `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Trivia Quiz](https://github.com/Nokse22/trivia-quiz) - Trivia game using questions from the Open Trivia database `#python` `#gtk4` `#libadwaita`.
```

- [TwoFun](https://gitlab.com/tabos/twofun/) - Simple touch based games inspired by "4 Spiele Reactor" `#c` `#gtk4` `#libadwaita`.

> - [Words!](https://codeberg.org/petsoi/words) - Game similar to similar to a popular word puzzle game where players try to guess a hidden word within six attempts. `#rust` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Words!](https://codeberg.org/petsoi/words) - Game where players try to guess a hidden word within six attempts. `#rust` `#gtk4` `#libadwaita`.
```

### Role-playing

> ### Simulation

Reviewer: ```suggestion
### Simulation

```

> - [Game of Life](https://flathub.org/apps/com.github.sixpounder.GameOfLife) - A simple Conway's game of life simulator for the Gnome desktop  `#rust` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Game of Life](https://flathub.org/apps/com.github.sixpounder.GameOfLife) - Simple Conway's game of life simulator for the GNOME desktop  `#rust` `#gtk4` `#libadwaita`.
```

> ### Sports

Reviewer: ```suggestion
### Sports

```

> - [tactics](https://gitlab.com/leesonwai/tactics) - A football lineup builder `#c` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Tactics](https://gitlab.com/leesonwai/tactics) - Football lineup builder `#c` `#gtk4` `#libadwaita`.
```

> ### Strategy

Reviewer: ```suggestion
### Strategy

```

> - [Freeciv](https://flathub.org/apps/org.freeciv.gtk4) - Empire-building strategy game inspired by the history of human civilization `#c` `#gtk4`.

Reviewer: ```suggestion
- [Freeciv GTK Client](https://flathub.org/apps/org.freeciv.gtk4) - Empire-building strategy game inspired by the history of human civilization `#c` `#gtk4`.
```
